front_end/ProcTime[] proc
int32 descriptID
int32[] left_featureMask
int32[] right_featureMask
sensor_msgs/Image leftDescriptors
sensor_msgs/Image rightDescriptors
